modules/sfp_bitcoinwhoswho.py

```diff
@@ -0,0 +1,133 @@
+    __name__ = 'sfp_bitcoinwhoswho'
+    meta = {
+        'name': "Bitcoin Who's Who",
+        'summary': "Check for malicious Bitcoin address agains BitcoinWhosWho database",
+        'flags': ["apikey"],
+        'useCases': ["Passive"],
+        'categories': ["Reputation Systems"],
+        'dataSource': {
+            'website': "https://bitcoinwhoswho.com/",
+            'model': "FREE_AUTH_LIMITED",
+            'references': [
+                "https://bitcoinwhoswho.com/api"
+            ],
+            'apiKeyInstructions': [
+                "Visit https://bitcoinwhoswho.com/signup",
+                "Register a free account",
+                "Verify your email and sign into your account",
+                "Visit https://bitcoinwhoswho.com/api/register and request an API Key",
+                "Wait for a few days, you'll receive it to your email"
+            ],
+            'favIcon': "https://bitcoinwhoswho.com/public/images/ico/favicon.ico",
+            'logo': "https://bitcoinwhoswho.com/public/images/logo2.png",
+            'description': (
+                "Bitcoin Who's Who is dedicated to profiling the extraordinary members of the bitcoin "
+                "ecosystem.Our goal is to help you verify a bitcoin address owner and avoid a bitcoin "
+                "scam or fraud."
+            ),
+        }
+    }
+
+    opts = {
+        'api_key': '',
+    }
+
+    optdescs = {
+        "api_key": "Bitcoin Who's Who API Key."
+    }
+
+    results = None
+
+    errorState = False
+
+    def setup(self, sfc, userOpts=None):
+        self.sf = sfc
+        self.results = self.tempStorage()
+
+        if userOpts:
+            self.opts.update(userOpts)
+
+    def watchedEvents(self):
+        return ["BITCOIN_ADDRESS"]
+
+    def producedEvents(self):
+        return ["MALICIOUS_BITCOIN_ADDRESS"]
+
+    def query(self, qry):
+        qs = urllib.parse.urlencode({"address": qry})
+        res = self.sf.fetchUrl(
+            f"https://bitcoinwhoswho.com/api/scam/{self.opts['api_key']}?{qs}",
+            timeout=self.opts["_fetchtimeout"],
+            useragent="SpiderFoot",
+        )
+
+        if res["content"] is None:
+            self.sf.info(f"No {self.meta['name']} info found for {qry}")
+            return None
+
+        try:
+            info = json.loads(res["content"])
+        except Exception as e:
+            self.sf.error(
+                f"Error processing JSON response from {self.meta['name']}: {e}", False
+            )
+            return None
+
+        return info
+
+    def emit(self, etype, data, pevent, notify=True):
+        evt = SpiderFootEvent(etype, data, self.__name__, pevent)
+        if notify:
+            self.notifyListeners(evt)
+        return evt
+
+    def generate_events(self, data, pevent):
+        if not isinstance(data, dict):
+            return
+
+        scams = data.get("scams", [])
+        if scams:
+            self.emit("MALICIOUS_BITCOIN_ADDRESS", pevent.data)
```

Comment:
Same as with other `MALICIOUS_` types, the content should be consistent with the others.